.github/actions/make/jvm-linux/action.yml

```diff
@@ -9,7 +9,14 @@ runs:
   steps:
       - uses: ./.github/actions/make
         with:
-          key: jvm-linux-${{ github.sha }}
+          key: jvm-linux
+          dependencies: |
+            Cargo.lock
+            keystore/**/*.rs
+            mls-provider/**/*.rs
+            crypto-macros/**/*.rs
+            crypto/**/*.rs
+            crypto-ffi/**/*.rs
```

Comment:
I am not in favor of this. We already started on this path in the `Makefile`. This is adding more of the same on top of that.